site/docs/components/border-layout/examples.mdx

```diff
@@ -70,7 +70,7 @@ Use the `verticalAlignment` and `horizontalAlignment` properties of the border i
 
 Use the `sticky` prop of the border item wrapper to define a region as sticky. When you set it to `true`, content in surrounding regions will scroll beneath a `sticky` Border Item.
```

Comment:
We could link to https://developer.mozilla.org/en-US/docs/Web/CSS/position#sticky to help